lib/css/modules/_showImages.scss

```diff
@@ -4,3 +4,49 @@ body:not(.res-showImages-displayImageCaptions) {
 		display: none;
 	}
 }
+
+.madeVisible {
+
+	position: relative;
+
+	.RESMediaControls {
+		opacity: 0;
+	}
+
+	&:hover .RESMediaControls {
+		opacity: .8;
+		transition: opacity 300ms;
+	}
+}
+
+.RESMediaControls {
+
+	position: absolute;
+	background: #fff;
+	z-index: 2;
+	bottom: 10px;
+	right: 10px;
+	border: solid 1px #eee;
+
+	span {
+		padding: 8px;
+		display: inline-block;
+		font: normal 14px/1 "Batch";
```

Comment:
How about putting `<span class="res-icon">` buttons into the control so you don't have to duplicate the icon setup css?